sdk/containerregistry/azure-containers-containerregistry-perf/src/main/java/com/azure/containers/containerregistry/perf/core/TestInputStream.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry.perf.core;
+
+import reactor.core.publisher.Flux;
+
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.util.Random;
+
+import static com.azure.containers.containerregistry.implementation.UtilsImpl.CHUNK_SIZE;
+
+public class TestInputStream extends InputStream {
+    private static final Random RANDOM = new Random(42);
```

Comment:
It looks like we can get rid of this and make it a local variable in the static initializer.